apps/challenges/views.py

```diff
@@ -3261,6 +3273,9 @@ def create_or_update_github_challenge(request, challenge_host_team_pk):
                         )
                         if serializer.is_valid():
                             serializer.save()
+                        else:
+                            error_messages = serializer.errors
+                            raise RuntimeError("Leaderboard validation failed")
```

Comment:
I am not sure why we want to raise the error here?

Shouldn't we aggregate all the error messages in one check and then display to the user? I think that is why we have `error_messages` variable, no?